### DISCUSSION TITLE
How can I remove the reference time in the transcript output file?

### DISCUSSION THREAD

**Discussion by palladinoandrea** [intel/openvino-plugins-ai-audacity#184] [General]
Is there a way to remove the reference time from a transcribed sentence? for example in my output file I find the line composed of: {initial time} {final time} sentence?

**Comment by RyanMetcalfeInt8** [comment]
Hi @palladinoandrea,

I'm not too sure. You might have better luck asking this question in primary Audacity repo ( https://github.com/audacity/audacity/discussions), as this export operation is a function of Audacity itself, and not these plugins. 

Thanks!
